packages/docs-app/src/examples/core-examples/labelExample.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as React from "react";
+
+import { InputGroup, Label, Switch } from "@blueprintjs/core";
+import { BaseExample, handleBooleanChange } from "@blueprintjs/docs-theme";
+
+export interface ILabelExampleState {
+    disabled: boolean;
+    helperText: boolean;
+    inline: boolean;
+}
+
+export class LabelExample extends BaseExample<ILabelExampleState> {
```

Comment:
given #2460, do we really want to add a `Label` example if we're gonna kill it for `FormGroup` soon?

Reply:
rewrite to use new `Example` component